py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert.py

```diff
@@ -4,81 +4,63 @@
 import pyarrow as pa
 
 from datetime import datetime
-from multiprocessing import Pool
-from pathlib import Path
+from pyarrow import fs
 
 from .configuration import Configuration
 
-# TODO this is fine for now, but maybe an environ variable?
-MULTIPROCESSING_POOL_SIZE = 4
-
-def gz_to_pyarrow(filename: Path, config: Configuration):
+def gz_to_pyarrow(filename: str, config: Configuration) -> pa.Table:
```

Comment:
read local file and convert. return either a `pa.Table` if successful or the `filename` if not.